Problem3/tc_array_duplicates.rb

```diff
@@ -0,0 +1,48 @@
+# File:  tc_array_duplicates.rb
+
+require_relative "array_duplicates.rb"
+require "test/unit"
+
+class TestArrayDuplicates < Test::Unit::TestCase
+	
+	def test_empty
+		
+		array_duplicates = ArrayDuplicates.new
+		numbers = Array[]
+		
+		returnedResult = array_duplicates.has_duplicates(numbers)		
+	
+		assert_equal(false, returnedResult)
+	end
+	
+	def test_no_duplicates
+		
+		array_duplicates = ArrayDuplicates.new
+		numbers = Array[5,3,2,1,4]
+		
+		returnedResult = array_duplicates.has_duplicates(numbers)		
+	
+		assert_equal(false, returnedResult)
+	end
+
+	
+	def test_only_duplicates
+		
+		array_duplicates = ArrayDuplicates.new
+		numbers = Array[5,5,5,5,5,5,5,5,5]
```

Comment:
Invalid test `Given an array of numbers having size N in which every number is between 1 and N` deci declaratia ta trebuie sa fie `numbers = [5, 5, 5, 5, 5]` (apropo nu ai nevoie de Array in fata)